engine/src/main/java/org/terasology/engine/TerasologyConstants.java

```diff
@@ -56,6 +56,8 @@
     /**
      * Name of the core gameplay module. The core gameplay module adds a minimal environment for some basic gameplay.
      */
+    //TODO: CSG moved out of the engine repo, so this should either be removed or reference the Builder Sample Gameplay
```

Comment:
I'm not sure what these constants are used for... would it make sense to change this to `buildersamplegameplay`?

Reply:
Some of the constants are surely still in use, but maybe not all. I think this should be fine to also change to BSG